opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
@antoviaque 

> I meant that the variables wouldn't have to be randomly generated 
I understand. With the approach you're recommending, we'd be using our regular code which does already generate random variables, but we could mock that part out for the tests.

> If we provide these values publicly, will it be secure anyway?

With the current approach, each person who loads the page would see a *unique* set of random secret values (and it would be served via HTTPS). So it's quite secure, though technically we could have recorded those values before serving them, so to be safe the end user should always re-generate their own secrets. This is just a case of being "[more] secure by default" vs. providing insecure defaults and telling people to change them. Not a big deal though.